src/codegen/src/main/scala/WrapperGenerator.scala

```diff
@@ -23,6 +23,8 @@ abstract class WrapperGenerator {
 
   def wrapperName(myClass: Class[_]): String
 
+  def modelWrapperName(myClass: Class[_], modelName: String): String
+
```

Comment:
does this need to also take in the modelName?

Reply:
the model wrapper needs to take in the model name because that is what it returns, it uses the class to determine whether to put an underscore at the beginning